_posts/2018-07-25-Relay-Networking-Deep-Dive.markdown

```diff
@@ -64,21 +75,25 @@ const fetchFunction = async (
 
   const data = await response.json();
 
+  // Mutations should throw when they have errors, making it easier
+  // for client code to react
   if (isMutation(request) && data.errors) {
     throw data;
   }
 
+  // We return the GraphQL response to update the Relay Environment
+  // which updates internal store where relay keeps its data
   return data;
 };
 ```
```

Comment:
I moved some of the documentation which was at the bottom of this code inline, so someone can read and directly compare